[2022-02-19 15:15:49,199] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cassandra_create_test_database_dag.create_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2022-02-19 15:15:49,213] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: cassandra_create_test_database_dag.create_table scheduled__2022-02-19T00:00:00+00:00 [queued]>
[2022-02-19 15:15:49,213] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 15:15:49,213] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-19 15:15:49,213] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 15:15:49,226] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): create_table> on 2022-02-19 00:00:00+00:00
[2022-02-19 15:15:49,231] {standard_task_runner.py:52} INFO - Started process 1941 to run task
[2022-02-19 15:15:49,235] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cassandra_create_test_database_dag', 'create_table', 'scheduled__2022-02-19T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/cassandra_create_test_database_dag.py', '--cfg-path', '/tmp/tmpgemd2bp6', '--error-file', '/tmp/tmpdefp4ueh']
[2022-02-19 15:15:49,235] {standard_task_runner.py:77} INFO - Job 14: Subtask create_table
[2022-02-19 15:15:49,292] {logging_mixin.py:109} INFO - Running <TaskInstance: cassandra_create_test_database_dag.create_table scheduled__2022-02-19T00:00:00+00:00 [running]> on host 38342be44bf7
[2022-02-19 15:15:49,346] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@***.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cassandra_create_test_database_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-19T00:00:00+00:00
[2022-02-19 15:15:49,358] {base.py:79} INFO - Using connection to: id: ***_default. Host: 172.20.0.11, Port: 9042, Schema: None, Login: ***, Password: ***, extra: {}
[2022-02-19 15:15:49,359] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/apache/***/hooks/***.py:123: DeprecationWarning: Legacy execution parameters will be removed in 4.0. Consider using execution profiles.
  self.cluster = Cluster(**conn_config)

[2022-02-19 15:15:49,367] {cassandra.py:131} WARNING - Downgrading core protocol version from 66 to 65 for 172.20.0.11:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/***/cluster.html#***.cluster.Cluster.protocol_version
[2022-02-19 15:15:49,370] {cassandra.py:131} WARNING - Downgrading core protocol version from 65 to 5 for 172.20.0.11:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/***/cluster.html#***.cluster.Cluster.protocol_version
[2022-02-19 15:15:49,372] {libevreactor.py:370} ERROR - Closing connection <LibevConnection(139645919633552) 172.20.0.11:9042> due to protocol error: Error from server: code=000a [Protocol error] message="Beta version of the protocol used (5/v5-beta), but USE_BETA flag is unset"
[2022-02-19 15:15:49,373] {cassandra.py:131} WARNING - Downgrading core protocol version from 5 to 4 for 172.20.0.11:9042. To avoid this, it is best practice to explicitly set Cluster(protocol_version) to the version supported by your cluster. http://datastax.github.io/python-driver/api/***/cluster.html#***.cluster.Cluster.protocol_version
[2022-02-19 15:15:49,705] {cassandra_create_test_database_dag.py:47} INFO - >>>>>>>>>>> Executed: Keyspace and Tables are created
[2022-02-19 15:15:49,706] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 15:15:49,723] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=***_create_test_database_dag, task_id=create_table, execution_date=20220219T000000, start_date=20220219T151549, end_date=20220219T151549
[2022-02-19 15:15:49,769] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 15:15:49,813] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
